### PR TITLE
r54842 - allowing Symfony builds to access the previous build var.

### PR DESCRIPTION
**Pull request by gregharvey**

### CODE REVIEW

File: symfony/fabfile.py [hunk 1]
```diff
@@ -160,6 +163,6 @@ def main(repo, repourl, branch, build, buildtype, siteroot, keepbuilds=10, url=N
   execute(common.Services.clear_varnish_cache, hosts=env.roledefs['app_all'])
 
   # Let's allow developers to perform some post-build actions if they need to
-  execute(common.Utils.perform_client_deploy_hook, repo, buildtype, build, buildtype, config, stage='post', hosts=env.roledefs['app_all'])
+  execute(common.Utils.perform_client_deploy_hook, repo, buildtype, build, buildtype, config, stage='post', build_hook_version="1", alias=None, site=None previous_build=previous_build, hosts=env.roledefs['app_all'])
```
Comment (EmlynK) [inline symfony/fabfile.py:166]: You've missed a comman between `site=None` and `previous_build=previous_build` :)

**Review by EmlynK — CHANGES_REQUESTED** [review]

File: symfony/fabfile.py (hunk 1 above)
Comment (gregharvey) [reply symfony/fabfile.py]: Well spotted, thanks! Should be OK now.

**Review by EmlynK — APPROVED** [review]
This looks good now! Although the `previous_build` variable isn't being used anywhere in the `perform_client_deploy_hook` function, I guess you're putting it there for future use? :)

**Review by EmlynK — APPROVED** [review]
Looks good to me!

**Pull request merged**